javaslang-benchmark/src/test/java/javaslang/collection/ArrayBenchmark.java

```diff
@@ -1,24 +1,42 @@
-package javaslang.benchmark.collection;
+package javaslang.collection;
 
-import javaslang.benchmark.JmhRunner;
+import javaslang.JmhRunner;
+import org.junit.Test;
 import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.infra.Blackhole;
 
 import java.util.*;
 
-import static javaslang.benchmark.JmhRunner.*;
+import static javaslang.JmhRunner.*;
 
 public class ArrayBenchmark {
+    static final Array<Class<?>> CLASSES = Array.of(
```

Comment:
I like this change. This allows us to selectively run groups of tests (useful during development) without having to comment out large areas of the code, which was very messy.